src/main/java/manufacturing/dao/jdbc/CarDaoJdbc.java

```diff
@@ -0,0 +1,201 @@
+package manufacturing.dao.jdbc;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import manufacturing.dao.CarDao;
+import manufacturing.exception.DataProcessingException;
+import manufacturing.injections.Dao;
+import manufacturing.model.Car;
+import manufacturing.model.Driver;
+import manufacturing.model.Manufacturer;
+import manufacturing.util.ConnectionUtil;
+
+@Dao
+public class CarDaoJdbc implements CarDao {
+
+    @Override
+    public Car create(Car car) {
+        String query = "INSERT INTO cars (manufacturer_id, model) VALUES (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement insertStatement =
+                        connection.prepareStatement(query, Statement.RETURN_GENERATED_KEYS)) {
+
+            insertStatement.setLong(1, car.getManufacturer().getId());
+            insertStatement.setString(2, car.getModel());
+            insertStatement.executeUpdate();
+            ResultSet resultSet = insertStatement.getGeneratedKeys();
+            if (resultSet.next()) {
+                car.setId(resultSet.getObject(1, Long.class));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Could not insert " + car, e);
+        }
+        return car;
+    }
+
+    @Override
+    public Optional<Car> get(Long id) {
+        String query = "SELECT cars.car_id AS car_id, cars.model, m.manufacturer_id AS m_id, "
+                + "m.manufacturer_name AS m_name, m.manufacturer_country AS m_country, "
+                + "d.driver_id AS d_id, d.driver_name AS d_name, d.license_number FROM cars "
+                + "LEFT JOIN cars_drivers cd ON cars.car_id = cd.car_id "
+                + "LEFT JOIN drivers d ON d.driver_id = cd.driver_id "
+                + "LEFT JOIN manufacturers m ON cars.manufacturer_id = m.manufacturer_id "
```

Comment:
i think that it's bad practice. You can do it using less joins. Try to rewrite it or implement private method with getting manufacturers by id and list of drivers

Reply:
This way is 12-13 times faster than repeating queries (we have tested it). It may be a little harder to read, but I think it is OK.

Reply:
Confirm the better performance of this solution